wpengine/faustjs
# 16: update lowest version of node to 18 since 16 is deprecating

## Description
Update dependency on node 16+ to 18+ since 16 is being deprecated.

## Testing
1. Ensure you have switched to the proper branch and pulled latest
2. Run `npm install` to install node modules based on node 18+
3. Test out the example projects to ensure there are no errors or failures in the build and display
4. Note: this does not include internal/legacy project
